db/migrate/20230616212113_remove_duplicate_user_id_from_likes.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDuplicateUserIdFromLikes < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :likes, :user_id
+  end
+end
```

Comment:
- [ ] Kindly remove this migration as it's throwing errors. you are trying to remove the user reference in like but this does not exist in likes.
<img width="779" alt="image" src="https://github.com/Itswali/blog_app/assets/79058364/ce0710c0-7fed-4cfd-a43c-3f0c4f4c73a8">